### DISCUSSION TITLE
External access via Cloudflare Tunnel - unable to add/edit links

### DISCUSSION THREAD

**Discussion by imamouse111** [JulianPrieber/littlelink-custom#768] [Issues]
Hello,

I'm able to add/edit links when accessing the dashboard from my home network via IP:port but when I'm accessing it via my Clouflare Tunnel domain, the add/edit link fields fail to load.

Internal Network
![Screenshot 2024-04-27 1 01 21 PM](https://github.com/LinkStackOrg/LinkStack/assets/4504206/649fb263-06c8-4f62-ab06-52fb8550331b)

External Cloudflare Tunnel
![Screenshot 2024-04-27 1 00 28 PM](https://github.com/LinkStackOrg/LinkStack/assets/4504206/7a1f13b7-de77-40c9-b1d0-0a42a985cfb7)

**Comment by JulianPrieber** [comment]
Enable force https in the config

**Reply by imamouse111** [reply]
Thanks for the reply.

HTTPS is already enabled on the domain. My URL begins with 'https://'. I also tried changing the tunnel's hostname service type from HTTP to HTTPS, but that breaks it and I get a 404 error page.

**Reply by JulianPrieber** [reply]
In your LinkStack config
